Documentation/ceph-pool-crd.md

```diff
@@ -31,6 +31,26 @@ spec:
   deviceClass: hdd
 ```
 
+#### Hybrid Storage Pools
+Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
+This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+    hybridStorage:
+      primaryDeviceClass: ssd
+      secondaryDeviceClass: hdd
+```
+Warning: Ensure that `primaryDeviceClass` and `secondaryDeviceClass` are valid and they have at least one OSD associated with them, else the crush rule re-compile will fail.
```

Comment:
Somehow, this is a piece of important information we need to highlight. So I think the warning makes sense. It's just too bad our doc does not auto-generate info/warning section like so docs do.

Reply:
we can do something like `> **Warning:**` which we usually do for warning in doc.

Reply:
Instead of warning how about `**IMPORTANT**`? If it's a warning, IMO it would be something that threatens cluster health, but in this case it would just prevent initial creation.